.github/workflows/build.yml

```diff
@@ -85,9 +85,45 @@ jobs:
           conda build .conda_mac --output-folder build
           echo "BUILD_PATH=$(pwd)/build" >> "$GITHUB_ENV"
 
-      # Test built conda package (Ubuntu and Windows)
-      - name: Test built conda package (Ubuntu and Windows)
-        if: matrix.os != 'macos-14'
+      # Test built conda package (Ubuntu)
+      - name: Test built conda package (Ubuntu)
+        if: matrix.os == 'ubuntu-22.04'
+        shell: bash -l {0}
+        run: |
+          echo "Current build path: $BUILD_PATH"
+          conda deactivate
+    
+          echo "Python executable before activating environment:"
+          which python
+          echo "Python version before activating environment:"
+          python --version
+          echo "Conda info before activating environment:"
+          conda info
+
+          echo "Creating and testing conda environment with sleap package..."
+          sudo apt-get update
+          sudo apt-get install libglapi-mesa libegl-mesa0 libegl1 libopengl0 libgl1-mesa-glx
+          conda create -y -n sleap_test -c file://$BUILD_PATH -c sleap/label/dev -c conda-forge -c nvidia -c anaconda sleap
+          conda activate sleap_test
```

Comment:
So it seems like this is the only difference between Windows/Ubuntu. For code re-usability, we could create more jobs like "List environment info" for before this block, "Create conda environment" for this block, and "Test environment" for the last block?

Or we can just merge it in because it is working

Reply:
This has been modified to 
1. Print build environment info
2. Build conda package (for each OS)
3. Print base environment info
4. Install OpenGL libraries on Ubuntu runner
5. Create conda environment (Ubuntu and Windows)
6. Create conda environment (Mac)
7. Test conda environment